.github/workflows/owners-validate.yml

```diff
@@ -1,20 +1,49 @@
-name: validate
+name: Run validation tests for owners
 on: [push, pull_request_target]
+
 jobs:
+  pr-info:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Query comment author repository permissions
```

Comment:
Seels to me that we are querying something different than comment author here IIUIC

Reply:
Fixed, good point!